content/dart/concept/list/terms/remove/remove.md

```diff
@@ -0,0 +1,56 @@
+
+
+---
+Title: 'List.remove()'
+Description: 'Removes first occurrence of a value from the list.'
+Subjects:
+  - 'Andriod Development'
+  - 'Dart'
```

Comment:
You should choose the subjects from this [list](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md).